pkgs/applications/networking/cluster/nomad/default.nix

```diff
@@ -12,14 +12,14 @@ buildGoPackage rec {
     owner = "hashicorp";
     repo = "nomad";
     inherit rev;
-    sha256 = "06s4irjz7k5qp895hvwv68xmr6mdq8bv91vhgh6rmx58my8jb68b";
+    sha256 = "0xj2asapl63rkr04lr7vdvdykmbsqvnglgfbf4wz3i2dn8bfwwam";
```

Comment:
The hash doesn't match for me:

    output path ‘/nix/store/l0n80d9p8hfv9m0asprskfwz97i3qpwr-nomad-v0.5.4-src’ has r:sha256 hash ‘0x7bi6wq7kpqv3wfhk5mqikj4hsb0f6lx867xz5l9cq3i39b5gj3’ when ‘0xj2asapl63rkr04lr7vdvdykmbsqvnglgfbf4wz3i2dn8bfwwam’ was expected